### PR TITLE
Added alignment properties for MetroDataGridRowHeader

### PR DESCRIPTION
**Pull request by chuuddo**
Now we can use style like this
```csharp
<Style x:Key="RowHeaderStyle" TargetType="DataGridRowHeader"
       BasedOn="{StaticResource MetroDataGridRowHeader}">
    <Setter Property="HorizontalContentAlignment" Value="Center" />
</Style>
```
```csharp
<DataGrid RowHeaderStyle="{StaticResource RowHeaderStyle}" />
```
screenshot_1

### CODE REVIEW

File: MahApps.Metro/Styles/Controls.DataGrid.xaml [hunk 1]
```diff
@@ -250,6 +250,10 @@
 
     <Style x:Key="MetroDataGridRowHeader"
            TargetType="{x:Type DataGridRowHeader}">
+        <Setter Property="HorizontalContentAlignment"
+                Value="Left" />
```
Comment (punker76) [inline MahApps.Metro/Styles/Controls.DataGrid.xaml]: the default should be Stretch

Comment (chuuddo) [inline MahApps.Metro/Styles/Controls.DataGrid.xaml]: @punker76  dont know, but in MetroDataGridColumnHeader default is Left.
```
<Style x:Key="MetroDataGridColumnHeader"
           TargetType="{x:Type DataGridColumnHeader}">
       ...
        <Setter Property="HorizontalContentAlignment"
                Value="Left" />
        <Setter Property="VerticalContentAlignment"
                Value="Center" />
```

Comment (punker76) [inline MahApps.Metro/Styles/Controls.DataGrid.xaml]: yeah i know, but the default before your changes is Stretch (it's the default if you doesn't say HorizontalAlignement="xyz")

Comment (chuuddo) [inline MahApps.Metro/Styles/Controls.DataGrid.xaml]: you are right

**Pull request merged**